<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPermission</class>
 <widget class="QWidget" name="FormPermission">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1210</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Права доступа</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabPermission">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabRoles">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Пользователи-&gt;Роли</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabObjects">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Роли-&gt;Права</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
